### PR TITLE
Update tests and pass RNGkind() to background process

### PR DESCRIPTION
**Pull request by cpsievert**

### CODE REVIEW

File: tests/testthat/recorded_tests/rmd-prerendered/tests/mytest-expected/001.json [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
     "bw_adjust": 1,
-    "n_breaks": "20"
+    "n_breaks": "20",
+    "n_breaks-selectized": ""
   },
   "output": {
     "eruptions": {
-      "src": "[image data sha1: 87a4f61ddd2d880d2d67631985bac379820e7732]",
+      "src": "[image data sha1: cebbfb5a3660a73cf0426844e057c40680785077]",
       "width": 738,
```
Comment (cpsievert) [inline tests/testthat/recorded_tests/rmd-prerendered/tests/mytest-expected/001.json]: Seems there's some randomness here that `ShinyDriver` can't capture? https://github.com/rstudio/shinytest/blob/b98f98c08a1b73a45768ab958c54cac6e1be08b5/tests/testthat/recorded_tests/rmd-prerendered/tests/mytest.R#L1

File: tests/testthat/recorded_tests/rmd-prerendered/tests/mytest-expected/001.json [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
+            },
+            "mapping": {
 
+            }
           }
+        ],
+        "dims": {
+          "width": 738,
+          "height": 626
         }
```
Comment (cpsievert) [inline tests/testthat/recorded_tests/rmd-prerendered/tests/mytest-expected/001.json]: Is this due to a change in shiny and/or ggplot2?

File: tests/testthat/test-list-widgets.R [hunk 3]
```diff
@@ -9,7 +9,7 @@ test_that("list input & output widgets", {
   expect_equal(
     sort(inputs),
     sort(c("action", "checkbox", "checkGroup", "date", "dates", "file",
-           "num", "radio", "select", "slider1", "slider2", "text"))
+           "num", "radio", "select", "select-selectized", "slider1", "slider2", "text"))
   )
```
Comment (cpsievert) [inline tests/testthat/test-list-widgets.R]: See rstudio/shiny@77afd73

File: tests/testthat/test-helper-rng.R [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# The default RNG kind changed in R > 3.6...the recorded tests
+# currently rely on randomness and are currently expecting
+# the default RNG kind from R 3.5
+# CRAN has suggested this as workaround for random tests
```
Comment (wch) [inline tests/testthat/test-helper-rng.R]: It'll help in the future to have a sentence saying that we can get rid of this and update all the expected results when we're no longer testing with R >=3.5 on Travis.

File: tests/testthat/test-list-widgets.R (hunk 3 above)
Comment (cpsievert) [reply tests/testthat/test-list-widgets.R]: Actually this was the culprit https://github.com/rstudio/shinytest/commit/eda9e7d48cbd5c6044d29160cafe70c82e8bd65c

Comment (wch) [reply tests/testthat/test-list-widgets.R]: Actually, it's from Shiny: https://github.com/rstudio/shiny/commit/6b6058

https://github.com/rstudio/shiny/issues/2396

**Pull request merged**